src/types/name.rs

```diff
@@ -76,9 +134,9 @@ impl<'a> From<&'a str> for NameAttribute<'a> {
     }
 }
 
-impl Name {
+impl<'a> Name<'a> {
```

Comment:
Now that this has a lifetime, we should add `Name::into_owned` as well.

Reply:
Done!